packages/suite-desktop-core/e2e/tests/trezor-connect/permissions.test.ts

```diff
@@ -0,0 +1,73 @@
+import TrezorConnect from '@trezor/connect-web';
+
+import { expect, test } from '../../support/fixtures';
+
+test.describe('TrezorConnect', { tag: ['@group=suite', '@desktopOnly'] }, () => {
+    test.use({ electronConf: { exposeConnectWs: true } });
+    test.beforeEach(async ({ onboardingPage, dashboardPage }) => {
+        await onboardingPage.completeOnboarding();
+        await dashboardPage.discoveryShouldFinish();
+        await test.step('Initialize TrezorConnect', async () => {
+            await TrezorConnect.init({
+                manifest: {
+                    appUrl: 'http://localhost:8080',
+                    email: '',
+                    appName: 'Tester',
+                },
+                coreMode: 'suite-desktop',
+                debug: true,
+            });
+        });
+    });
+
+    test('Permissions - add and forget', async ({
+        settingsPage,
+        connectModal,
+        trezorUserEnvLink,
+        page,
+        devicePrompt,
+    }) => {
+        await settingsPage.navigateTo('connect');
+
+        await page.getByTestId('@settings/connect-apps/no-apps').waitFor({ state: 'visible' });
+
+        TrezorConnect.getAddress({
+            path: "m/44'/0'/0'/0/0",
+            coin: 'btc',
+        });
+
+        await expect(connectModal.processParagraph).toHaveText('Process: node');
```

Comment:
```suggestion
        await expect(connectModal.processParagraph).toHaveText('node');
```
Process label is gone in new design